app/javascript/components/home/JoinOurWelcomingCommunitySection.jsx

```diff
@@ -2,12 +2,15 @@ import React from 'react';
 
 const JoinOurWelcomingCommunitySection = () => {
     return (
-        <>
+        <p>
             Attend our monthly meetup or start up a conversation in Slack to begin interacting with
             our community of over 350 women and non-binary Rubyists. Here at WNB.rb, we value safety
-            above all else; that is why we have a code of conduct [LINK] to to prevent gatekeeping
-            and other unsavory behavior.
-        </>
+            above all else; that is why we have a{' '}
+            <a href="https://tinyurl.com/wnb-rb-coc" target="_blank" rel="noopener noreferrer">
+                code of conduct
+            </a>{' '}
```

Comment:
what does `{' '}` do here? this is new to me!

Reply:
It adds a space into the text! Most text editors will cut any extra spaces at the end of the line, but we need that space there so there's a space between the word "a" and the word "code". Prettier will automatically insert one of these if it has to format the code in such a way that it eliminates a space!